### ISSUE TITLE
Add max. profit to stats page

### ISSUE THREAD

**Issue by RHavar** [moneypot/webserver#146]

**Comment by csmcanarney** [comment]
Do you mean the maximum profit for each game on that game's page or the all-time highest net profit on each player's page?

**Comment by kungfuant** [comment]
I think it means adding the current maximum profit to https://www.moneypot.com/stats so that others can scrape it.

**Comment by RHavar** [comment]
Yeah, what kunguant said, it's just for https://thebitcoinstrip.com/ to scrape. Also we should add the sites current bankroll for them as well

**Comment by csmcanarney** [comment]
What do you think about adding a simple API for that instead or in addition? If people are going to scrape the site anyways, might as well make it a little easier to do and cut down on bandwidth IMO.
